src/services/AccountService.ts

```diff
@@ -86,4 +86,12 @@ export class AccountService {
   }
 }
 
+const getLocalStorageItem = (key: string): string | null => {
+  try {
+    return localStorage.getItem(key);
```

Comment:
A comment like `May fail due to security policies in Chrome's incognito mode` would be helpful for future readers.